azure-pipelines.release.yml

```diff
@@ -43,8 +43,8 @@ steps:
   #   displayName: npm run generate-version-files
 
   - script: |
-      node common/scripts/install-run-rush.js build --production --verbose
-    displayName: rush build (Create production build)
+      yarn build --production
```

Comment:
Is the default behavior now to have verbose output? (as opposed to Rush's default very minimal output)

Reply:
yup!